Sources/PrimerSDK/Classes/User Interface/Components/PrimerResultComponentView.swift

```diff
@@ -27,36 +27,35 @@ class PrimerResultComponentView: PrimerView {
     }
     
     fileprivate func initialize() {
-        addSubview(stackView)
+        addSubview(verticalStackView)
         
-        stackView.axis = .vertical
-        stackView.spacing = 20.0
-        stackView.alignment = .center
+        verticalStackView.axis = .vertical
+        verticalStackView.spacing = 20.0
+        verticalStackView.alignment = .center
         
-        stackView.translatesAutoresizingMaskIntoConstraints = false
-        stackView.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
-        stackView.topAnchor.constraint(greaterThanOrEqualTo: topAnchor).isActive = true
-        stackView.trailingAnchor.constraint(equalTo: trailingAnchor).isActive = true
-        stackView.bottomAnchor.constraint(lessThanOrEqualTo: bottomAnchor).isActive = true
-        stackView.centerYAnchor.constraint(equalTo: centerYAnchor).isActive = true
+        verticalStackView.translatesAutoresizingMaskIntoConstraints = false
+        verticalStackView.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
+        verticalStackView.topAnchor.constraint(greaterThanOrEqualTo: topAnchor).isActive = true
+        verticalStackView.trailingAnchor.constraint(equalTo: trailingAnchor).isActive = true
+        verticalStackView.bottomAnchor.constraint(lessThanOrEqualTo: bottomAnchor).isActive = true
+        verticalStackView.centerYAnchor.constraint(equalTo: centerYAnchor).isActive = true
         
         if let imageView = imageView {
-            stackView.addArrangedSubview(imageView)
+            verticalStackView.addArrangedSubview(imageView)
         }
         
         if let message = message {
             let messageLabel = UILabel()
             messageLabel.text = message
-            messageLabel.accessibilityIdentifier = "result_component_view_message_label"
```

Comment:
Don't we need this for UI tests?

Reply:
We rely on the image now but I'll add it back - thanks for catching it!